users/views.py

```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
+import requests, jwt
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+from django.conf  import settings
+from datetime     import datetime, timedelta
+
+from users.models import User
```

Comment:
datetime은 built in package이므로 맨 윗단에서 선언해 주세요.
```suggestion
from datetime     import datetime, timedelta

from django.views import View
from django.http  import JsonResponse
from django.conf  import settings

from users.models import User
```